proposals/proposalTypes/TimelockProposal.sol

```diff
@@ -76,21 +79,22 @@ abstract contract TimelockProposal is Proposal {
             vm.prank(proposerAddress);
 
             // Perform the low-level call
-            (bool success, ) = payable(timelockAddress).call(scheduleCalldata);
-
-            // Check if the call was successful
-            require(success, "Call to schedule timelock failed");
+            bytes memory returndata = address(payable(timelockAddress)).functionCall(scheduleCalldata);
```

Comment:
```
    function functionCallWithValue(address target, bytes memory data, uint256 value) internal returns (bytes memory) {
        (bool success, bytes memory returndata) = target.call{value: value}(data);
        return verifyCallResultFromTarget(target, success, returndata);
    }


/**
     * @dev Tool to verify that a low level call to smart-contract was successful, and reverts if the target
     * was not a contract or bubbling up the revert reason (falling back to {FailedInnerCall}) in case of an
     * unsuccessful call.
     */
    function verifyCallResultFromTarget(
        address target,
        bool success,
        bytes memory returndata
    ) internal view returns (bytes memory) {
        if (!success) {
            _revert(returndata);
        } else {
            // only check if target is a contract if the call was successful and the return data is empty
            // otherwise we already know that it was a contract
            if (returndata.length == 0 && target.code.length == 0) {
                revert AddressEmptyCode(target);
            }
            return returndata;
        }
    }
    
         * @dev Reverts with returndata if present. Otherwise reverts with {FailedInnerCall}.
     */
    function _revert(bytes memory returndata) private pure {
        // Look for revert reason and bubble it up if present
        if (returndata.length > 0) {
            // The easiest way to bubble the revert reason is using memory via assembly
            /// @solidity memory-safe-assembly
            assembly {
                let returndata_size := mload(returndata)
                revert(add(32, returndata), returndata_size)
            }
        } else {
            revert FailedInnerCall();
        }
    }
    ```
    
   `if (!success) { _revert(returndata);` is not doing the same as 
  `require(condition, abi.encodePacked("error message executing in timelock: ", getRevertData(message))`
   `